google/cloud/datastore/batch.py

```diff
@@ -290,7 +290,7 @@ def _assign_entity_to_pb(entity_pb, entity):
 
     Helper method for ``Batch.put``.
 
-    :type entity_pb: :class:`google.cloud.datastore._generated.entity_pb2.Entity`
+    :type entity_pb: :class:`.datastore._generated.entity_pb2.Entity`
```

Comment:
I'd say if we are going to use it, we should go whole hawg and make it look like a "normal" relative path:  you and I are not going to be the only Python programmers who trip over it.